dashboards/traefik/traefik.json

```diff
@@ -282,7 +282,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\nlatest(traefik_tls_certs_not_after)*1000 AS 'Expiration Date'\nFACET prometheus_server, instance, cn AS 'Common Name', sans AS 'Subject Alternate Name', serial\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write'"
+                "query": "FROM Metric SELECT\nlatest(traefik_tls_certs_not_after)*1000 AS 'Expiration Date'\nFACET concat(prometheus_server,' - ',instance) AS 'Instance', cn AS 'Common Name', sans AS 'Sunject Alternate Name', serial\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} )"
```

Comment:
typo here:

`Sunject Alternate Name` should be `Subject Alternate Name`

Reply:
Fixed. Thanks.